static/locales/en.client.json

```diff
@@ -1,779 +1,748 @@
-      "TotalSize":"The total size of all data in this document, excluding attachments.",
-      "Updates":"Updates every 5 minutes.",
-      "AttachmentsSize": "Attachments Size",
-      "DataSize": "Data Size",
-      "Usage": "Usage",
-      "LimitContactSiteOwner": "Contact the site owner to upgrade the plan to raise limits.",
-      "UpgradeLinkText": "start your 30-day free trial of the Pro plan.",
-      "ForHigherLimits": "For higher limits, ",
-      "StatusMessageApproachingLimit": "This document is {{- link}} free plan limits.",
-      "StatusMessageGracePeriod": "Document limits {{- link}}.",
-      "StatusMessageGracePeriodElse": "Document limits {{- link}}. In {{gracePeriodDays}} days, this document will be read-only.",
-      "StatusMessageDeleteOnly": "This document {{- link}} free plan limits and is now read-only, but you can delete rows.",
-      "Rows": "Rows"
-    },
-    "ViewConfigTab": {
-      "UnmarkOnDemandTitle": "Unmark table On-Demand?",
-      "UnmarkOnDemandButton": "Unmark On-Demand",
-      "UnmarkOnDemandText": "If you unmark table {{- table}}' as On-Demand, its data will be loaded into the calculation engine and will be available for use in formulas. For a big table, this may greatly increase load times.{{- br}}{{-br}}Changing this setting will reload the document for all users.",
-      "MakeOnDemandTitle": "Make table On-Demand?",
-      "MakeOnDemandButton": "Make On-Demand",
-      "MakeOnDemandText": "If you make table {{table}} On-Demand, its data will no longer be loaded into the calculation engine and will not be available for use in formulas. It will remain available for viewing and editing.",
-      "AdvancedSettings": "Advanced settings",
-      "BigTablesMayBeMarked": "Big tables may be marked as \"on-demand\" to avoid loading them into the data engine.",
-      "Form": "Form",
-      "Compact": "Compact",
-      "Blocks": "Blocks",
-      "EditCardLayout": "Edit Card Layout",
-      "PluginColon": "Plugin: ",
-      "SectionColon": "Section: "
-    },
-    "Drafts": { 
-      "UndoDiscard":"Undo discard",
-      "RestoreLastEdit":"Restore last edit"
-    },
-    "duplicatePage": {
-      "DoesNotCopyData":"Note that this does not copy data, but creates another view of the same data.",
-      "DuplicatePageName":"Duplicate page {{pageName}}"
-    },
-    "GristDoc": {
-      "ImportFromFile":"Import from file",
-      "AddedNewLinkedSection":"Added new linked section to view {{viewName}}",
-      "SavedLinkedSectionIn":"Saved linked section {{title}} in view {{name}}"
-    },
-    "Importer": {
-      "UpdateExistingRecords":"Update existing records",
-      "MergeRowsThatMatch":"Merge rows that match these fields:",
-      "SelectFieldsToMatch":"Select fields to match on"
-    },
-    "PluginScreen": {
-      "ImportFailed":"Import failed: "
-    },
-    "RecordLayout": {
-      "UpdatingRecordLayout":"Updating record layout."
-    },
-    "RecordLayoutEditor": {
-      "AddField":"Add Field",
```

Comment:
Those keys as still used.

Reply:
They are not really removed. I removed the folder key (in this case `components`) to have a flatter file. The new RecordLayoutEditor translations look like this :

```javascript
  "RecordLayoutEditor": {
    "Add Field": "Add Field",
    "Create New Field": "Create New Field",
    "Show field {{- label}}": "Show field {{- label}}",
    "Save Layout": "Save Layout",
    "Cancel": "Cancel"
  }
  ```
Of course the file has been updated and there the key named `AddField` is replaced by `Add Field`